pkg/config/config.go

```diff
@@ -168,6 +181,12 @@ func (c *Config) applyMissingNames() {
 		host.name = key
 		c.Hosts[key] = host
 	}
+	for key, _ := range c.Templates {
```

Comment:
should omit 2nd value from range; this loop is equivalent to `for key := range ...`